packages/editor/src/extensions/table/component.tsx

```diff
@@ -125,15 +126,19 @@ function TableRowToolbar(props: TableToolbarProps) {
       const currentRow = findSelectedDOMNode(editor, ["tableRow"]);
       if (!currentRow) return;
 
-      const pos = getPosition(rowToolsRef.current, {
-        location: "left",
-        target: currentRow,
-        align: "start",
-        xOffset: -5,
-        yOffset: -3
-      });
-      rowToolsRef.current.style.top = `${pos.top}px`;
-      rowToolsRef.current.style.left = `${pos.left}px`;
+      try {
+        const pos = getPosition(rowToolsRef.current, {
```

Comment:
I think it might be a better idea to handle this at the `getPosition` level instead of wrapping this in a try-catch.

Reply:
We can add a check to handle the `getBoundingClientRect` issue.

Currently `getPosition` function expects an `HTMLElement`; instead we can change the type to `Node` which includes all sorts of nodes. This will force us to check and make sure the element is actually an `HTMLElement`.

Reply:
resolved by 3075afa6ae6e8f061ae0f8601d1c915e14510895